curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md

```diff
@@ -27,35 +27,31 @@ You should have a `main` element inside your `body` element.
 ```js
 assert.equal(document.querySelector('body :first-child').tagName, 'MAIN');
 ```
-
-Your `main` element should be the only child of the `body` element.
+Your main element should be the only child of the body element.
```

Comment:
```suggestion

Your main element should be the only child of the body element.
```
empty lines around code fences are needed

Reply:
you also removed backticks, do not do that

Reply:
@ilenia-magoni sorry for the inconvenience from my side , I am new to this